features/step_definitions/ocm/ocm.rb

```diff
@@ -1,18 +1,27 @@
-Given /^I open ocm portal as an? #{WORD} user$/ do |usertype|
+Given /^I open ocm portal with #{WORD}$/ do |usertype|
   base_rules = BushSlicer::WebConsoleExecutor::RULES_DIR
   snippets_dir = BushSlicer::WebConsoleExecutor::SNIPPETS_DIR
+  portals = YAML.load_file(expand_private_path("config/credentials/ocm.yaml"))
```

Comment:
as we chatted, lets use config.yaml in private repo and avoid reading external file